### PR TITLE
uar-168 update feature check

### PR DESCRIPTION
**Pull request by AndreKramer7654**
[JIRA link](https://companieshouse.atlassian.net/browse/UAR-168)

### Change description
Add a service level check to disable any new update related pages when feature flag is not explicitly enabled.

### Work checklist

- [ ] Tests added where applicable
- [ ] UI changes meet accessibility criteria

### Merge instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this, this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

**Comment by markpit** [comment]
The sonar report shows 1 code smell needs fixing and unit test code coverage needs to be > 80% preferably in the 90s

File: src/config/index.ts [hunk 1]
```diff
@@ -32,6 +32,8 @@ export const PAYMENT_PAID = "paid";
 export const SHOW_SERVICE_OFFLINE_PAGE = getEnvironmentValue("SHOW_SERVICE_OFFLINE_PAGE");
 export const FEATURE_FLAG_ENABLE_SAVE_AND_RESUME_17102022 = getEnvironmentValue("FEATURE_FLAG_ENABLE_SAVE_AND_RESUME_17102022", "false");
 export const FEATURE_FLAG_ENABLE_TRUSTS_WEB = <any>getEnvironmentValue('FEATURE_FLAG_ENABLE_TRUSTS_WEB_07112022', 'false');
+export const FEATURE_FLAG_ENABLE_ROE_UPDATE = getEnvironmentValue('FEATURE_FLAG_ENABLE_ROE_UPDATE_24112022', 'false');
+export const FEATURE_FLAG_ENABLE_ROE_REMOVE = getEnvironmentValue('FEATURE_FLAG_ENABLE_ROE_REMOVE_24112022', 'false');
```
Comment (jkayyala) [inline src/config/index.ts:36]: The feature flag change is not needed as the change has already been merged...

Comment (AndreKramer7654) [reply src/config/index.ts]: merged now

**Comment by AndreKramer7654** [comment]
> The sonar report shows 1 code smell needs fixing and unit test code coverage needs to be > 80% preferably in the 90s

The sonar report is from 4 days ago and I can't seem to trigger a rerun with my pushes. How do I trigger a new analysis?

**Review by jkayyala — APPROVED** [review]

**Review by markpit — APPROVED** [review]

**Pull request merged**